mappings/net/minecraft/server/ServerCommandHandler.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/class_426 net/minecraft/server/ServerCommandHandler
+	FIELD field_1686 logger Ljava/util/logging/Logger;
+	FIELD field_1687 minecraftServer Lnet/minecraft/server/MinecraftServer;
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 minecraftServer
+	METHOD method_1411 executeCommand (Lnet/minecraft/class_38;)V
+		ARG 1 command
+	METHOD method_1412 parseInt (Ljava/lang/String;I)I
+		ARG 1 intString
```

Comment:
probably only [string](https://maven.fabricmc.net/docs/yarn-1.20-pre4+build.2/net/minecraft/util/math/MathHelper.html#parseInt(java.lang.String,int)) in yarn, seems like the method has understandably been moved to MathHelper